contrib/completion/bash/docker

```diff
@@ -1027,6 +1027,7 @@ _docker_run() {
 		--net
 		--pid
 		--publish -p
+		--read-bps-device
```

Comment:
I agree with @tiborvass here.
Most of the cgroup names are pretty sane, but this one is a bit backwards to me. I'd prefer to provide a better name, especially since this will be shared by other platforms.
This is just a nit...

We could probably even go a bit further and say `device-read-rate` and have it accept a user-friendly value  (e.g. `50mbps` or something instead of `52428800`).